google/cloud/security/inventory/pipeline_builder.py

```diff
@@ -90,8 +90,8 @@ def _get_api(self, api_name):
                 else:
                     api = api_class(self.global_configs,
                                     version=api_version)
-            except api_errors.ApiExecutionError as e:
-                LOGGER.error('Failed to execute API %s, v=%s\n%s',
+            except api_errors.ApiInitializationError as e:
+                LOGGER.error('Failed to initialize API %s, v=%s\n%s',
                              api_class_name, api_version, e)
                 raise api_errors.ApiInitializationError(e)
```

Comment:
This can just be the following, since you are re-raising the same exception now:

raise